src/lib.rs

```diff
@@ -219,7 +222,7 @@ impl Discord {
 	///
 	/// The token will automatically be prefixed with "Bot ".
 	pub fn from_bot_token(token: &str) -> Result<Discord> {
-		Ok(Discord::from_token_raw(format!("Bot {}", token.trim())))
+		Ok(Discord::from_token_raw("Bot ".to_string() + token.trim()))
```

Comment:
Yeah, it's not so big a deal in this single-concatenation instance, but it gets more pronounced when you have many more things you're sticking together.